src/Api/Resources/ItemLastSeenResource.php

```diff
@@ -32,10 +33,26 @@ public function __construct(Request $request, ApiResponse $response)
      */
     public function index():Response
     {
-        $items = $this->request->get("items", 4);
+        $items = $this->request->get("items", 20);
```

Comment:
Warum wird der Standard-Wert hier so hoch gesetzt? Ich denke hier ist 4 auch weiterhin ein sinnvoller Wert, um zu große Datenmengen zu vermeiden.

Reply:
Wir wollten es einheitlich haben mit dem Wert aus dem Service. Ich werde es wieder anpassen und in Ceres den Rest-Call anpassen.